src/store/users/selectors.ts

```diff
@@ -0,0 +1,7 @@
+import { AppState } from '../types';
+import { Collaborator } from '../../@types';
+
+type State = Pick<AppState, 'users'>;
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Should we disable or modify this rule at a global level? It seems like we almost always override it.